### PR TITLE
Query update Animals table and Queries

### PR DESCRIPTION
**Pull request by marvin-nyalik**
In this branch: 
- I added data to the animals table. Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.
2query1.

 Rollback the species and verify the species columns went back to initial state. 
2query2
- Update the animals table by setting the species column to digimon for all animals that have a name ending in mon. Update the animals table by setting the species column to pokemon for all animals that don't have species already set. Verify that changes persist after commit.
2query3

- Now, take a deep breath and... Inside a transaction delete all records in the animals table.
2QUERY4 

- Then rollback the transaction.
2QUERY5

-Delete all animals born after Jan 1st, 2022. Create a savepoint for the transaction.
2query7

- Update all animals' weight to be their weight multiplied by -1.
2query8

- Rollback to the savepoint
2query9

- How many animals are there
2query10

-How many animals have tried to escape
2QUERY11

-What is the average weight of animals?
2query12

-Who escapes the most, neutered or not neutered animals?
2query13

-What is the minimum and maximum weight of each type of animal?
2QUERY14

-What is the average number of escape attempts per animal type of those born between 1990 and 2000?
2QUERY15

Let me know of any required changes.

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -7,4 +7,62 @@ SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
...
+ROLLBACK TO after_2022;
+
+UPDATE animals
+SET weight_kg = weight_kg * -1
+WHERE weight_kg < 0;
+
+COMMIT;
+
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals WHERE escape_attempts > 0;
```
Comment (LionRouge1) [inline queries.sql]: - [ ] Nice work creating queries to answer the questions :clap: However, note that the `have never` expression is expressed in POSTGRESSQL by `= 0` operation, therefore your query for the second question (**How many animals have never tried to escape?**) should look like this :point_down: 
  ```sql
     SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
  ```

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @marvin-nyalik, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Update `data.sql` file to add additional data :heavy_check_mark: 
- Write queries to answer the questions :heavy_check_mark: 
- Add screenshots of the queries results in the PR :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by marvin-nyalik** [comment]
MAKING CHANGES:

How many animals have never tried to escape?
haveNever

**Review by prolajumokeoni — APPROVED** [review]
Hi @marvin-nyalik 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- Changes requested have been implemented   ✅ 
  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**